Flow improvement (#43)
githedgehog/docs: docs/vlab/running.md

## Initialize VLAB

> First, initialize Fabricator  by running `hhfab init --dev`.This command create the `fab.yaml` file, which is the main configuration file for the fabric. This command supports several customization options that are listed in the output of `hhfab init --help`.

Reviewer: ```suggestion
First, initialize Fabricator by running `hhfab init --dev`. This command creates the `fab.yaml` file, which is the main configuration file for the fabric. This command supports several customization options that are listed in the output of `hhfab init --help`.
```

21:27:16 INF >>> mclagServers=2 eslagServers=2 unbundledServers=1 bundledServers=1
21:27:16 INF Generated wiring file name=vlab.generated.yaml
```

> [Click Here](#build-the-installer-and-start-vlab) to start VLAB, or see the next section for customizing the topology.

Reviewer: I don't like this, we don't know where the link is taking us. Instead of “Click here” I'd use something like `You can jump straight to [the instructions to start the VLAB](<link>), or see ...`.

Author: agreed

## Build the Installer and Start VLAB

> To build and start the virtual machines use `hhfab vlab up`. For successive runs use the `--kill-stale` flag to ensure that any virtual machines from a previous run are gone. `hhfab vlab up` runs in the foreground and does not return. Running in the foreground makes stopping all VLAB vms a simple `ctrl + c`.

Reviewer: ```suggestion
To build and start the virtual machines, use `hhfab vlab up`. For successive runs, use the `--kill-stale` flag to ensure that any virtual machines from a previous run are gone. `hhfab vlab up` runs in the foreground and does not return, which allows you to stop all VLAB VMs by simply pressing `Ctrl + C`.
```

## Reset VLAB

> If VLAB is currently running, press `ctrl + c` to stop it. To reset VLAB and start over run `hhfab init -f` which will force overwrite your existing configuration in `fab.yaml`.

Reviewer: ```suggestion
If VLAB is currently running, press `Ctrl + C` to stop it. To reset VLAB and start over run `hhfab init -f`. This option forces the process to overwrite your existing configuration in `fab.yaml`.
```
